src/core/qgsmapsettings.h

```diff
@@ -154,12 +155,15 @@ class CORE_EXPORT QgsMapSettings
 
     //! Check whether the map settings are valid and can be used for rendering
     bool hasValidSettings() const;
-    //! Return the actual extent derived from requested extent that takes takes output image size into account
+    //! Return the extent of the visible area, see visiblePolygon()
     QgsRectangle visibleExtent() const;
-    //! Return the visible area as a polygon (may be rotated)
+    //! Return the visible area as a polygon. (may be rotated)
+    //! The visible area is derived from requested extent, rotation
+    //! and output image size.
```

Comment:
similarly, this should have a "@see visibleExtent" tag to link to the related method